libs/ngrx-accelerator/src/lib/store-connector/onecx-actions.ts

```diff
@@ -0,0 +1,10 @@
+import { createActionGroup, props } from '@ngrx/store'
+
+export const OneCxActions = createActionGroup({
+  source: 'OneCX',
+  events: {
+    navigated: props<{
+      event: undefined | unknown
```

Comment:
I guess there is a reason why the type here is not PayloadNavigatedEvent?

Reply:
see PM